lib/tools/adb-commands.js

```diff
@@ -735,28 +735,36 @@ methods.setWifiAndData = async function ({wifi, data}, isEmulator = false) {
 
 /**
  * Change the state of animation on the device under test.
+ * Animation on the device is controlled by the following global properties:
+ * [ANIMATOR_DURATION_SCALE]{@link https://developer.android.com/reference/android/provider/Settings.Global.html#ANIMATOR_DURATION_SCALE},
+ * [TRANSITION_ANIMATION_SCALE]{@link https://developer.android.com/reference/android/provider/Settings.Global.html#TRANSITION_ANIMATION_SCALE},
+ * [WINDOW_ANIMATION_SCALE]{@link https://developer.android.com/reference/android/provider/Settings.Global.html#WINDOW_ANIMATION_SCALE}.
+ * This method sets all this properties to 0.0 to disable (1.0 to enable) animation.
+ *
+ * Turning off animation might be useful to improve stability
+ * and reduce tests execution time.
  *
  * @param {boolean} on - True to enable and false to disable it.
  */
 methods.setAnimationState = async function (on) {
-  await this.shell(['am', 'broadcast', '-a', `${ANIMATION_SETTING_ACTION}`,
-                    '-n', `${SETTINGS_HELPER_ID}/${ANIMATION_SETTING_RECEIVER}`,
+  await this.shell(['am', 'broadcast', '-a', ANIMATION_SETTING_ACTION,
+                    '-n', ANIMATION_SETTING_RECEIVER,
                     '--es', 'setstatus', on ? 'enable' : 'disable']);
 };
 
 /**
  * Check the state of animation on the device under test.
  *
  * @return {boolean} True if at least one of animation scale settings
- *                   is not equal to zero.
+ *                   is not equal to "0.0".
  */
 methods.isAnimationOn = async function () {
   let animator_duration_scale = await this.getSetting('global', 'animator_duration_scale');
   let transition_animation_scale = await this.getSetting('global', 'transition_animation_scale');
   let window_animation_scale = await this.getSetting('global', 'window_animation_scale');
-  return (parseFloat(animator_duration_scale, 10) !== 0 ||
-          parseFloat(transition_animation_scale, 10) !== 0 ||
-          parseFloat(window_animation_scale, 10) !== 0);
+  return (animator_duration_scale !== "0.0" ||
```

Comment:
👍

Reply:
can also be `!_.some([animator_duration_scale, transition_animation_scale, window_animation_scale], (x) => x === '0.0')`